src/components/Modal.js

```diff
@@ -52,6 +53,8 @@ const Modal = (props) => {
             style={modalStyle}
             animationIn={animationIn}
             animationOut={animationOut}
+            useNativeDriver
+            useNativeDriverForBackdrop
```

Comment:
I'm not sure but these are possibly causing some flickering animations on iOS. I commented out these lines and things smoothed out again.

https://github.com/react-native-modal/react-native-modal#the-modal-flashes-in-a-weird-way-when-animating

Reply:
Went on the `react-native-modal` Github page, and found a couple ([[1](https://github.com/react-native-modal/react-native-modal/issues/268)] [[2](https://github.com/maxs15/react-native-modalbox/issues/265)]) of issues on this topic.

There seems to have some workarounds if more people complain of performance issues, and we want to test out these props again.

Removed it for now.